src/services/data-provider.js

```diff
@@ -29,7 +29,9 @@ export default {
     return httpClient(`${endpoint.url}?${stringify(query)}`).then(
       ({ json }) => ({
         data: endpoint.getData(json.data),
-        total: json.data.pageInfo.totalRecord,
+        total: json.data.pageInfo
+          ? json.data.pageInfo.totalRecord
+          : json.data.length,
```

Comment:
will `data.length` always exist whenever `data.pageInfo` does not...?

Reply:
ah ofcos we always return an array for multiple records..

Reply:
Yes, that's the assumption. But its still flawed because its based on an assumption.